gtfs/gtfs_utils/gtfs_utils/s3.py

```diff
@@ -90,3 +94,40 @@ def fetch_remote_file(remote_file_key: str,
     logging.debug(f'Finished file download (key="{remote_file_key}", local path="{local_file_full_path}")')
     return True
     # TODO: log file size
+
+
+def is_enough_disk_space(remote_file_keys: List[str],
+                         crud: S3Crud,
+                         max_download_size: float = None,
+                         local_dir: str = None,
+                         suppress_errors: bool = True) -> bool:
+    """
+    Check if there is enough free disk space in the dest dir to download the files from the S3
+    returns True if there is enough space
+    if not enough space returns False or raise IOError if suppress is set to false
+
+    :param remote_file_keys: a list of S3 file keys
+    :param crud: S3Crud object
+    :param max_download_size: limit the download size (in bytes)
+    :param local_dir: the local dir the files will be saved in
+    :param suppress_errors: when files are to big if False raise IOError, else return False
+    :return: wether there is enough space or not
+    """
+    if local_dir is None:
+        local_dir = configuration.files.full_paths.gtfs_feeds
```

Comment:
Use it as a default value instead of setting it here (also for `max_download_size`)

Reply:
I didn't want to deal with setting mutable values as default values.